t/t3100-flux-in-flux.t

```diff
@@ -39,9 +39,8 @@ test_expect_success "flux --parent works in subinstance" '
 '
 
 test_expect_success "flux --parent --parent works in subinstance" '
-	id=$(flux submit \
-		flux start ${ARGS} \
-		flux start ${ARGS} flux --parent --parent kvs put test=ok) &&
+	id=$(flux batch -n1 --wrap \
+		flux run ${ARGS} flux --parent --parent kvs put test=ok) &&
```

Comment:
Is a `flux start` missing here. That is to simulate a depth of 2 for the `flux kvs put`, should this be:
```
id=$(flux batch -n1 --wrap \
    flux run ${ARGS} flux start flux --parent --parent kvs put test=ok)
```

Reply:
There's the outer test instance form `test_under_flux` (level 0), then the `flux batch` instance (level1), then the `flux start` instance (level2).  So I think it's right?

Reply:
I just see `flux run ${ARGS} flux --parent --parent kvs put test=ok` -- where am I missing the `flux start`?
(In the diff I see two `flux start` calls removed, replaced with one `flux batch` and one `flux run`) Sorry if I'm missing something obvious, sometimes hard to see via a diff...)

Reply:
Oh I do apologize - I was reading your "shouldn't it be" code snippet thinking it was the original.  You are quite right!